crates/relayer/src/chain/penumbra/chain.rs

```diff
@@ -522,10 +522,13 @@ impl ChainEndpoint for PenumbraChain {
             }
         };
 
+        // No support for custom registry.json files in Hermes yet.
+        let registry_path: Option<camino::Utf8PathBuf> = None;
```

Comment:
Is there a way to avoid pulling in the `camino` crate as a workspace dep?

Reply:
Is there a reason to? It's useful

Reply:
Ah, my intent wasn't clear. I should have asked: "Is explicitly declaring the type `Option<camino::Utf8PathBuf>` the minimal change necessary to satisfy?"

Reply:
`registry_path` is an `Option<impl AsRef<Utf8Path>>` and `String` implements `AsRef<Utf8Path>` so we can:

```rust

        let svc = rt
            .block_on(ViewServer::load_or_initialize(
                view_file,
                // No support for custom registry.json files in Hermes yet.
                Option::<String>::None,
                &fvk,
                config.grpc_addr.clone().into(),
            ))
            .map_err(|e| Error::temp_penumbra_error(e.to_string()))?;
```

Reply:
Thanks, that's exactly what I was reaching for.